draft-irtf-cfrg-vdaf.md

```diff
@@ -1615,8 +1615,26 @@ def run_flp(Flp, inp: Vec[Flp.Field], num_shares: Unsigned):
     # Prover generates the proof.
     proof = Flp.prove(inp, prove_rand, joint_rand)
 
-    # Verifier queries the input and proof.
-    verifier = Flp.query(inp, proof, query_rand, joint_rand, num_shares)
+    # Shard the input and the proof.
+    input_shares = linear_secret_share(inp, num_shares, Flp.Field)
```

Comment:
It's pretty self-explanatory what `linear_secret_share` does, but it's a good idea to at least define it somewhere in prose for completeness. One option is to stick in the list of functions in {{conventions-and-definitions}}. In particular we should define what we mean by "linear" and "secret share".

Reply:
Sounds good. I think I should also be calling it "additive secret sharing", as it's more specific.

Reply:
Agreed. These are sometimes used interchangeably, but it's good for the standard to be specific.